constants/discordRoles.ts

```diff
@@ -0,0 +1,7 @@
+const DISCORD_ROLES = {
+  UNVERIFIED: "unverified",
+};
+
+module.exports = {
+  DISCORD_ROLES,
+};
```

Comment:
- instead of module.exports can we export the object directly?
```js
export const DISCORD_ROLES = {
    ...
}
```

Reply:
Exported the object directly as you asked.